test/spec/dataviews/category-dataview/categories-collection.spec.js

```diff
@@ -0,0 +1,50 @@
+var Backbone = require('backbone');
+var CategoriesCollection = require('../../../../src/dataviews/category-dataview/categories-collection');
+
+describe('categories-collection', function () {
+  var aggregationModel;
+  var collection;
+
+  beforeEach(function () {
+    aggregationModel = new Backbone.Model({
+      aggregation: 'count'
+    });
+
+    collection = new CategoriesCollection(null, {
+      aggregationModel: aggregationModel
+    });
+  });
+
+  it('reset should filter null when reset', function () {
```

Comment:
To make this test a bit more clear/explicit I'd split it into two tests:

```
describe('.reset', function () {
  it('should NOT filter null values when categories are aggregated by "count"') { ... }
  it('should filter null values when categories are NOT aggregated by "count"') { ... }
});
```

Consider the PR approved after this.